query.go

```diff
@@ -0,0 +1,67 @@
+package lytics
```

Comment:
Probably not super important if we're looking to get this out ASAP, but usually we have tests for these. Mark or I can add them later too.